### PR TITLE
issue #85 integration of logging

### PR DESCRIPTION
**Pull request by opsdep**

### CODE REVIEW

**Review by akorosov — DISMISSED** [review]
Good!

File: geospaas_harvesting/ingesters.py [hunk 1]
```diff
@@ -603,7 +603,9 @@ def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
         " addresses or http addresses, not for ftp protocol")
 
         # Open file with Nansat
-        nansat_object = Nansat(nansat_filename(dataset_info), **nansat_options)
+        nansat_object = Nansat(nansat_filename(dataset_info),
+                               log_level=logging.getLogger("geospaas_harvesting").level,
```
Comment (aperrin66) [inline geospaas_harvesting/ingesters.py]:
```suggestion
                               log_level=self.LOGGER.level,
```

Comment (opsdep) [reply geospaas_harvesting/ingesters.py]: I have encountered that before, and now once again I debug that:
image

It does not work that way!

Comment (aperrin66) [reply geospaas_harvesting/ingesters.py]: Right, you need to use `self.LOGGER.getEffectiveLevel()`

Comment (opsdep) [reply geospaas_harvesting/ingesters.py]: done in 7b3a040

**Review by aperrin66 — APPROVED** [review]

**Review by akorosov — APPROVED** [review]

**Pull request merged**